romana-install/config.devstack.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+  tasks:
+    - add_host: name="{{ item }}" groups="devstack_nodes,devstack_controller"
+      with_items: groups.controller
+      when: stack_type == "devstack"
+    - add_host: name="{{ item }}" groups="devstack_nodes,devstack_computes"
```

Comment:
Does this mean that the devstack controller is not part of the devstack-computes group? Meaning, no guests will ever run there?